packages/beacon-node/src/chain/blocks/verifyBlocksSanityChecks.ts

```diff
@@ -20,22 +23,20 @@ import {BlockInput, BlockInputType, ImportBlockOpts, BlobSidecarValidation} from
  *   - Not finalized slot
  *   - Not already known
  */
-export function verifyBlocksSanityChecks(
-  chain: {forkChoice: IForkChoice; clock: IClock; config: ChainForkConfig},
+export async function verifyBlocksSanityChecks(
```

Comment:
will remove async from here as this is not needed here (was trying something but finally moved to verifyBlocksDataAvailablity in verifyBlock parallel verify calls with execution